addon/components/radio-button.js

```diff
@@ -1,9 +1,12 @@
 import Ember from 'ember';
+import layout from '../templates/components/radio-button';
```

Comment:
@raycohen This is what ember-cli gives you by default when you generate a component: `ember g component radio-button`. I'm not sure if you can use the addon's namespace. I can try.